src/server/rest/v1/service/RegistrationTokenService.ts

```diff
@@ -81,21 +81,8 @@ export default class RegistrationTokenService {
   }
 
   static async handleUpdateRegistrationToken(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
-    // Filter
-    const filteredRequest = RegistrationTokenSecurity.filterRegistrationTokenUpdateRequest(req.body);
-    UtilsService.assertIdIsProvided(action, filteredRequest.id, MODULE_NAME, 'handleUpdateRegistrationToken', req.user);
-    // Get Token
-    const registrationToken = await RegistrationTokenStorage.getRegistrationToken(req.user.tenantID, filteredRequest.id);
-    UtilsService.assertObjectExists(action, registrationToken, `Token ID '${filteredRequest.id}' does not exist`,
-      MODULE_NAME, 'handleUpdateRegistrationToken', req.user);
-    if (Utils.isComponentActiveFromToken(req.user, TenantComponents.ORGANIZATION)) {
-      // Check Site Area
-      const siteArea = await SiteAreaStorage.getSiteArea(req.user.tenantID, filteredRequest.siteAreaID);
-      UtilsService.assertObjectExists(action, siteArea, `Site Area ID '${filteredRequest.siteAreaID}' does not exist`,
-        MODULE_NAME, 'handleUpdateRegistrationToken', req.user);
-    }
     // Check Auth
-    if (!Authorizations.canUpdateRegistrationToken(req.user, registrationToken.siteArea?.siteID)) {
+    if (!Authorizations.canUpdateRegistrationToken(req.user, req.body.siteAreaID)) {
```

Comment:
What if the organization is not active in the tenant, should we pass the site area ID?
We cannot forbid the usage of token because the org is inactive.
Have you tested it with a tenant without an Org? Also check in the UI that Site Area is not showing up in that case.

Reply:
I let it as is as we will need to handle it soon after refactoring authorizations. NB: in the UI we don't see site area if org component is not active